### PR TITLE
draft: added changelog and posthog

### PR DESCRIPTION
**Pull request by Its-Alamin-H**

### CODE REVIEW

**Comment by 0xSage** [comment]
Is this ready to be reviewed? IF so, pls resolve conflicts, undraft the PR,  and tag a reviewer

**Comment by Its-Alamin-H** [comment]
> Is this ready to be reviewed? IF so, pls resolve conflicts, undraft the PR, and tag a reviewer

conflicts resolved

File: docs/docusaurus.config.js [hunk 1]
```diff
@@ -6,24 +6,26 @@ const darkCodeTheme = require("prism-react-renderer/themes/dracula");
...
   // Set the /<baseUrl>/ pathname under which your site is served
   // For GitHub pages deployment, it is often '/<projectName>/'
   baseUrl: "/",
-
   // GitHub pages deployment config.
   // If you aren't using GitHub pages, you don't need these.
   organizationName: "janhq", // Usually your GitHub org/user name.
   projectName: "jan", // Usually your repo name.
-
+  onBrokenLinks: "ignore",
```
Comment (0xSage) [inline docs/docusaurus.config.js:26]: No, we shouldn't leave in broken links

File: docs/docusaurus.config.js [hunk 2]
```diff
@@ -6,24 +6,26 @@ const darkCodeTheme = require("prism-react-renderer/themes/dracula");
 
 /** @type {import('@docusaurus/types').Config} */
 const config = {
+  title: "Jan Documentation",
+  tagline: "Self-hosted AI inference platform",
+  favicon: "img/favicon.ico",
+  // Set the production url of your site here
+  url: "https://docs.jan.ai",
```
Comment (0xSage) [inline docs/docusaurus.config.js:13]: This is no longer the correct link. It's just jan.ai

File: docs/docusaurus.config.js [hunk 3]
```diff
@@ -6,24 +6,26 @@ const darkCodeTheme = require("prism-react-renderer/themes/dracula");
 
 /** @type {import('@docusaurus/types').Config} */
 const config = {
+  title: "Jan Documentation",
+  tagline: "Self-hosted AI inference platform",
```
Comment (0xSage) [inline docs/docusaurus.config.js:10]: Run your own AI

File: docs/blog/authors.yml [hunk 4]
```diff
@@ -0,0 +1,34 @@
+Daniel:
```
Comment (0xSage) [inline docs/blog/authors.yml:1]: No, take out this entire page. I don't want hardcoded contributor content when Github already shows who the contributors are

File: docs/blog/Jan v10.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
+---
```
Comment (0xSage) [inline docs/blog/Jan v10.md:1]: rename folder from "blog" to "changelog" please

File: docs/blog/Jan v10.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+title: JanAI v1.0
```
Comment (0xSage) [inline docs/blog/Jan v10.md:2]: This is wrong. 
1. We're "Jan" not "JanAI"
2. The version is incorrect. Are you tracking the releases here: https://github.com/janhq/jan/releases

File: docs/blog/Jan v10.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+description: First changelog
+slug: jan v1.0
+authors: [hiento09, Louis, Daniel, automaticcat, namvuong, Nicole]
```
Comment (0xSage) [inline docs/blog/Jan v10.md:5]: skip this field

File: docs/blog/Jan v10.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+hide_table_of_contents: false
+---
+
+Stay Updated with JanAI: Explore Our Latest Changelog.
```
Comment (0xSage) [inline docs/blog/Jan v10.md:9]: "Jan"

File: docs/blog/Jan v10.md [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+
+<!--truncate-->
+
+### 🎉 We're thrilled to introduce the first version of Jan Web!
```
Comment (0xSage) [inline docs/blog/Jan v10.md:13]: Is this content just a placeholder? We're not doing Jan Web. 

Let's structure changelog content like this: https://docs.docker.com/desktop/release-notes/

File: docs/blog/Jan v10.md [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+
+What is Jan Web, you ask? It’s pretty much a web version of our Jan Mobile, that lets you generate amazing art via Open Source AI Models as well as engaging with the most popular AI Chatbots in the market. The best part of this? It’s totally free!
```
Comment (0xSage) [inline docs/blog/Jan v10.md:15]: Sounds overly promotional. Changelogs should be factual

File: docs/docusaurus.config.js [hunk 11]
```diff
@@ -184,6 +248,16 @@ const config = {
             title: "Docs",
             items: [
               {
+                label: "Developer",
```
Comment (0xSage) [inline docs/docusaurus.config.js:251]: where does this go? out of date?

File: docs/docusaurus.config.js [hunk 12]
```diff
@@ -164,6 +219,15 @@ const config = {
         style: "dark",
         links: [
           {
+            title: "Documentation",
+            items: [
+              {
+                label: "Getting Started",
```
Comment (0xSage) [inline docs/docusaurus.config.js:225]: where does this go? out of date?

File: docs/docusaurus.config.js [hunk 13]
```diff
@@ -247,4 +334,18 @@ const config = {
     }),
 };
 
+// Integrating PostHog product analytics with Docusaurus
+
+config.plugins = [
+  [
+    "posthog-docusaurus",
+    {
+      apiKey: process.env.POSTHOG_API_KEY || "DEV",
```
Comment (0xSage) [inline docs/docusaurus.config.js:343]: Are you coordinating with @hiento09 on what the API key is/where we're storing it?

File: docs/docusaurus.config.js [hunk 14]
```diff
@@ -138,6 +170,29 @@ const config = {
           // Navbar right
           {
             type: "docSidebar",
+
```
Comment (0xSage) [inline docs/docusaurus.config.js:173]: this is out of date

File: docs/docusaurus.config.js [hunk 15]
```diff
@@ -138,6 +170,29 @@ const config = {
...
+            sidebarId: "integrationsSidebar",
+            position: "left",
+            label: "Integrations",
+          },
+          {
+            position: "left",
```
Comment (0xSage) [inline docs/docusaurus.config.js:179]: this is out of date

**Review by 0xSage — CHANGES_REQUESTED** [review]

**Pull request closed without merging**